shell/platform/android/io/flutter/embedding/android/FlutterView.java

```diff
@@ -981,11 +990,46 @@ public void revertImageView() {
       Log.v(TAG, "Tried to revert the image view, but no previous surface was used.");
       return;
     }
-    flutterImageView.detachFromRenderer();
     renderSurface = previousRenderSurface;
     previousRenderSurface = null;
```

Comment:
The fact that there's a single instance variable for the previous surface seems weird... can you render an image view over an image view?  should we keep a stack of surfaces?

Reply:
ack. 
> can you render an image view over an image view? 

No.

Only `FlutterSurfaceView/FlutterTextureView` <-> `FlutterImage`, so a stack isn't needed as there could be 2 surfaces max.